spinalcordtoolbox/image.py

```diff
@@ -364,7 +364,6 @@ def __deepcopy__(self, memo):
     def copy(self, image=None):
         from copy import deepcopy
         if image is not None:
-            self.im_file = deepcopy(image.im_file)
```

Comment:
Just realizing I missed copying the affine here, since `self.im_file` is being replaced by `self.affine`.